starter-code/scripts/article.js

```diff
@@ -58,5 +62,11 @@ Article.fetchAll = function() {
     // cache it in localStorage so we can skip the server call next time,
     // then load all the data into Article.all with the .loadAll function above,
     // and then render the index page.
+    // getting the code from the server //
+    $.getJSON('data/hackerIpsum.json').then(function(rawData){
+      Article.loadAll(rawData);
+      localStorage.rawData = JSON.stringify(rawData);
```

Comment:
This is how you normally set something in local storage : localStorage.setItem('rawData', JSON.stringify(data));